srcrepo.go

```diff
@@ -292,7 +292,7 @@ func gitClone(logger logrus.FieldLogger, to string, url string, bare bool) error
 	}
 
 	// Move into the final location.
-	return os.Rename(tmpDir, to)
+	return copyDirectory(tmpDir, to)
```

Comment:
Do you know what error happens when an `os.Rename` fails because the src/dest are on different drives? Perhaps this should first attempt to do the `os.Rename` then fall back to the copyDirectory+delete method.

Reply:
Since I don't know the error I'm not sure if this is related, but I saw a failure in the rename call:

time="2020-11-24T13:07:54Z" level=warning msg="Failed to install Go from binary package. Trying to install from source." error="rename C:\\Users\\jenkins\\.gvm\\versions\\go1.14.7.windows.amd64.tmp\\go C:\\Users\\jenkins\\.gvm\\versions\\go1.14.7.windows.amd64: Access is denied." version=1.14.7`